app/views/layouts/application.html.haml

```diff
@@ -66,10 +66,15 @@
               %div.govuk-grid-column-one-third
                 %h2.govuk-heading-m
                   = t('footer.for_job_seekers')
-                %ul.govuk-footer__inline-list
-                  %li.govuk-footer__inline-list-item
+                %ul.govuk-footer__inline-list.hide-bullet-points
+                  %li.govuk-footer-list-item
                     =link_to t('footer.search_teaching_vacancies'), root_url, class: 'govuk-footer__link'
-
+                  - if authenticated? == false && JobseekerAccountsFeature.enabled?
```

Comment:
I would go for `!authenticated?` instead of `authenticated? == false`, and perhaps also flip them around to make the feature flag more obvious:

```ruby
- if JobseekerAccountsFeature.enabled? && !authenticated?
```